content/source/docs/enterprise/migrate/workspaces.html.md

```diff
@@ -31,120 +31,91 @@ Make sure you have all of the following:
 - A TFE user account which is a member of your organization's owners team, so you can create workspaces.
 - A [user API token][user-token] for your TFE user account. (Organization and team tokens will not work; the token must be associated with an individual user.)
 
-    In your shell, set an `ATLAS_TOKEN` environment variable with your API token as the value.
+    In your shell, set a `TFE_TOKEN` environment variable with your API token as the value.
 
     ``` bash
-    export ATLAS_TOKEN=<USER TOKEN>
+    export TFE_TOKEN=<USER TOKEN>
```

Comment:
Same as above, point users to the credentials stanza in the config file instead of an env var.